### ISSUE TITLE
I'm sorry

### ISSUE THREAD

**Issue by Dayuanshao** [XunpengYi/Diff-Retinex#4]
I'm a beginner and while preparing to train the IDA module, I saw you mentioning that the reflection image of the training set obtained through the TDA model should be placed in R. However, I only found the image generated from the inference test set, not the image generated from the training set using the model. How can I do that?
